packages/php-db-import-export/src/Backend/Snowflake/ToFinalTable/SqlBuilder.php

```diff
@@ -352,29 +352,31 @@ public function getUpdateWithPkCommand(
         }
 
         $columnsComparisonSql = [];
-        if ($importOptions->compareAllColumnsInNativeTable()) {
-            $columnsComparisonSql = array_map(
-                static function ($columnName) {
-                    return sprintf(
-                        '"dest".%s IS DISTINCT FROM "src".%s',
-                        SnowflakeQuote::quoteSingleIdentifier($columnName),
-                        SnowflakeQuote::quoteSingleIdentifier($columnName),
-                    );
-                },
-                $stagingTableDefinition->getColumnsNames(),
-            );
-        } elseif ($importOptions->isNullManipulationEnabled()) {
-            // update only changed rows - mysql TIMESTAMP ON UPDATE behaviour simulation
-            $columnsComparisonSql = array_map(
-                static function ($columnName) {
-                    return sprintf(
-                        'COALESCE(TO_VARCHAR("dest".%s), \'\') != COALESCE("src".%s, \'\')',
-                        SnowflakeQuote::quoteSingleIdentifier($columnName),
-                        SnowflakeQuote::quoteSingleIdentifier($columnName),
-                    );
-                },
-                $stagingTableDefinition->getColumnsNames(),
-            );
+        switch (true) {
+            case $importOptions->isNullManipulationEnabled():
+                // update only changed rows - mysql TIMESTAMP ON UPDATE behaviour simulation
+                $columnsComparisonSql = array_map(
+                    static function ($columnName) {
+                        return sprintf(
+                            'COALESCE(TO_VARCHAR("dest".%s), \'\') != COALESCE("src".%s, \'\')',
+                            SnowflakeQuote::quoteSingleIdentifier($columnName),
+                            SnowflakeQuote::quoteSingleIdentifier($columnName),
+                        );
+                    },
+                    $stagingTableDefinition->getColumnsNames(),
+                );
+                break;
+            case $importOptions->compareAllColumnsInNativeTable():
+                $columnsComparisonSql = array_map(
+                    static function ($columnName) {
+                        return sprintf(
+                            '"dest".%s IS DISTINCT FROM "src".%s',
+                            SnowflakeQuote::quoteSingleIdentifier($columnName),
+                            SnowflakeQuote::quoteSingleIdentifier($columnName),
+                        );
+                    },
+                    $stagingTableDefinition->getColumnsNames(),
+                );
```

Comment:
Co se stane pokud to nebude ani jedno?

Reply:
toto https://github.com/keboola/storage-backend/blob/a3b24940d3b63a41f8d5190ab834ce43fa2d122f/packages/php-db-import-export/src/Backend/Snowflake/ToFinalTable/SqlBuilder.php#L388

Reply:
není tam pak porovnání, je to dosavadní chování pro tabulky s native types